dev/com.ibm.ws.repository/src/com/ibm/ws/repository/resources/internal/EsaResourceImpl.java

```diff
@@ -256,7 +256,8 @@ private void addVersionDisplayString() {
             minVersion.startsWith("14.") ||
             minVersion.startsWith("15.") ||
             minVersion.startsWith("16.") ||
-            minVersion.startsWith("17.")) {
+            minVersion.startsWith("17.") ||
+            minVersion.startsWith("18.")) {
```

Comment:
I believe line 260 should be completely removed and line 259 should read:
```
minVersion.startsWith("17.")) {
```
@jhanders34 would you please confirm that?

Reply:
updated